### PR TITLE
Feature/29692

### PR DESCRIPTION
**Pull request by kjugi**
+ Add libs helper in `gulpfile.js` 
+ Change links to CSS and JS links 
+ Move it to other dir

https://dev.snowdog.pro/issues/29692

### CODE REVIEW

File: components/blocks/_preview.hbs [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
     <link media="all" rel="stylesheet" href="{{ path '/css/styles.css' }}">
-    <script src="{{ path '/js/svg4everybody.js' }}"></script>
-    <script src="{{ path '/js/lazysizes.min.js' }}"></script>
+    <script src="{{ libs 'svg4everybody.js' }}"></script>
+    <script src="{{ libs 'lazysizes.min.js' }}"></script>
     <script src="{{ path '/js/main.js' }}"></script>
-    <script src="{{ path '/js/jquery.js' }}"></script>
-    <script src="{{ path '/js/slick.min.js'}}"></script>
+    <script src="{{ libs 'jquery.js' }}"></script>
+    <script src="{{ libs 'slick.min.js'}}"></script>
```
Comment (Igloczek) [inline components/blocks/_preview.hbs]: Let's replace all scripts with unminified versions.

File: docs/styles.scss [hunk 2]
```diff
@@ -2,8 +2,8 @@
 @import '../components/general/fonts/fonts';
 
 // libs
-@import 'lib/normalize';
-@import 'lib/slick';
+@import '../components/general/libs/scss/normalize';
```
Comment (Igloczek) [inline docs/styles.scss]: Let's update to 7.0.0

File: gulpfile.js [hunk 3]
```diff
@@ -131,6 +131,10 @@ hbsEngine.handlebars.registerHelper('ifCond', function (v1, operator, v2, option
     }
 });
 
+hbsEngine.handlebars.registerHelper('libs', file => {
```
Comment (Igloczek) [inline gulpfile.js]: We can make it even shorter
```js
hbsEngine.handlebars.registerHelper('libs', file => `/components/raw/libs/${file}`);
```

File: docs/styles.scss (hunk 2 above)
Comment (kjugi) [inline docs/styles.scss]: updated

File: gulpfile.js (hunk 3 above)
Comment (kjugi) [inline gulpfile.js]: changed

File: components/blocks/_preview.hbs (hunk 1 above)
Comment (kjugi) [inline components/blocks/_preview.hbs]: actual libs replaced

Comment (Igloczek) [inline components/blocks/_preview.hbs]: **unminified**, not **minified**

Comment (kjugi) [inline components/blocks/_preview.hbs]: oh, my bad. Now it's unminified at all JS libs

**Pull request merged**